src/components/Book.js

```diff
@@ -27,13 +28,15 @@ const Book = (props) => {
       <section className="progress">
         <div className="progress-stat">
           <div className="progress-circle" />
-          <div className="progress-percent" />
-          <div className="progress-state" />
+          <div className="progress-status">
+            <div className="progress-percent">100%</div>
+            <div className="progress-state">Completed</div>
+          </div>
         </div>
```

Comment:
- [ ] In order to improve your UI design and to more closely match Zipline's design, it would be better to reduce the page's left, right, and top margin sizes.
<img width="732" alt="image" src="https://user-images.githubusercontent.com/77190203/181967084-b023ab31-f64b-4cc0-8718-10cf9ff2dcad.png">